apps/protocol-frontend/src/hooks/useDaoUserCreate.ts

```diff
@@ -43,7 +43,11 @@ export const useDaoUserCreate = () => {
             description: `${
               creatingNewDao === true
                 ? 'You have successfully created a new DAO.'
-                : 'You have successfully joined the DAO as a recruit.'
+                : `Successfully added to the DAO as a ${
+                    newDaoUser.membershipStatus
+                      ? newDaoUser.membershipStatus.toLowerCase()
+                      : 'recruit'
+                  }.`
```

Comment:
This is a bit complex it will be clearer if this turnary is abstracted to a variable

Reply:
Great callout!